diff --git a/node_modules/nexus-prisma/dist-cjs/generator/models/javascript.js b/node_modules/nexus-prisma/dist-cjs/generator/models/javascript.js
index af28df0..bd6e764 100644
--- a/node_modules/nexus-prisma/dist-cjs/generator/models/javascript.js
+++ b/node_modules/nexus-prisma/dist-cjs/generator/models/javascript.js
@@ -70,9 +70,9 @@ function createModuleSpec(params) {
       const gentimeSettings = ${JSON.stringify(gentimeSettings.data, null, 2)}
 
       const dmmf = getPrismaClientDmmf({
-        require: () => require('${gentimeSettings.data.prismaClientImportId}'),
+        require: () => require(${JSON.stringify(gentimeSettings.data.prismaClientImportId)}),
         importId: gentimeSettings.prismaClientImportId,
-        importIdResolved: require.resolve('${gentimeSettings.data.prismaClientImportId}')
+        importIdResolved: require.resolve(${JSON.stringify(gentimeSettings.data.prismaClientImportId)})
       })
 
       const models = ModelsGenerator.JS.createNexusTypeDefConfigurations(dmmf, {
